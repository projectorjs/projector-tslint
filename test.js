// @flow
"use strict";

const tslint = require("./");
const glob = require("glob");

test("success", () => {
  const files = glob.sync("./example/no-errors/src/*.ts");

  return tslint.run({
    files,
    configPath: "./example/no-errors/tslint.json"
  });
});

test("failure", () => {
  expect.assertions(1);
  const files = glob.sync("./example/with-errors/src/*.ts");

  return tslint
    .run({
      files,
      configPath: "./example/with-errors/tslint.json"
    })
    .catch(err => {
      expect(err.length).toMatchSnapshot();
    });
});
