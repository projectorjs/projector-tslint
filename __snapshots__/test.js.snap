// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`failure 1`] = `2`;
